draft-js-plugins-editor/CHANGELOG.md

```diff
@@ -14,6 +14,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - `decorators` don't decorate plugins anymore.
 
+### Changed
+
+- *Breaking change* The `plugins` and `decorators` editor props need to be an Immutable.List()
```

Comment:
I am not sure yet how to deal with decorators set outside of the editor, need to think about that a bit